doxygen.cmake

```diff
@@ -184,6 +184,28 @@ MACRO(_SETUP_PROJECT_DOCUMENTATION_FINALIZE)
       SET(DOXYGEN_HEADER_NAME "header.html")
     ENDIF()
 
+    # Find doxytag files
+    # To ignore this list of tag files, add to doc/Doxyfile.extra.in
+    # TAGFILES =
+    STRING(REPLACE "," ";" PKG_REQUIRES "${_PKG_CONFIG_REQUIRES}")
```

Comment:
`PKG_CONFIG_REQUIRES` does not include the packages that were added as compile only dependencies. I don't think this feature is used a lot and afaik there is no variables that contains every `pkg-config` dependencies. Maybe this could be added.

Reply:
Does it make sense to link the doc to compile time dependencies ?

Reply:
You are right. This does not make much sense.